core/src/main/resources/db/changelog/changesets/2024_08_28_in_person_kits.yaml

```diff
@@ -10,7 +10,7 @@ databaseChangeLog:
         - addColumn:
             tableName: kit_request
             columns:
-              - column: { name: kit_barcode, type: text, constraints: { nullable: true } }
+              - column: { name: kit_label, type: text, constraints: { nullable: true } }
```

Comment:
normally we would never modify an existing changelog, but this hasn't been run against any live database yet and i'd rather not introduce a superfluous changeset